src/unrepl/make_blob.clj

```diff
@@ -56,36 +56,58 @@
         regular s))
     (str sb)))
 
-(defn- gen-blob [session-actions]
+(defn- gen-blob [session-actions shade?]
   (let [template (slurp (io/resource "unrepl/blob-template.clj"))
-        shaded-code-sb (StringBuilder.)
-        shaded-libs (shade/shade 'unrepl.repl
-                      {:writer (fn [_ ^String code] (.append shaded-code-sb code))
-                       :except [#"clojure\..*" 'unrepl.core]})
-        code (-> template
-               (str/replace "unrepl.repl"
-                 (str (shaded-libs 'unrepl.repl)))
-               (str/replace "<BLOB-PAYLOAD>" (str shaded-code-sb)))]
+        code     (if shade?
+                   (let [shaded-code-sb (StringBuilder.)
+                         shaded-libs (shade/shade 'unrepl.repl
+                                                  {:writer (fn [_ ^String code] (.append shaded-code-sb code))
+                                                   :except [#"clojure\..*" 'unrepl.core]})]
```

Comment:
Shouldn't `:except #".*"` prevent shading?

Reply:
ok I think https://github.com/Unrepl/unrepl/commit/41eea93c1e4eb372169ccd7aa5ed191c362c8046 is relevant

Reply:
I will have a look later today. Now `:except #".*"` will work. However it feels very ugly to run this whole non-trivial machinery to do … nothing. I'd rather have shade doesn't read anything, but only feed it the list a relevant things.

Reply:
As a sidenote: the `--noshade` option should probably be something like `--except`.